### ISSUE TITLE
Add check API

### ISSUE THREAD

**Issue by dmitriim** [catalyst/moodle-tool_objectfs#480]
As this plugin can be a critical path to the inf set up, would be useful to add some health check using check API.

Candidates:

- [ ] proxyrangerequests is anbled, but "Test range request failed"
- [ ] A number of missing files are growing 
- [ ] ....
- [ ] ....

**Comment by brendanheywood** [comment]
+1 but lets not worry too much about getting them all done in this issue, we can split them up into different checks and issues

**Comment by dmitriim** [comment]
Yes. We can throw more ideas here and then tackle them separately.

**Comment by brendanheywood** [comment]
+1 to have a super basic check which is that it can both connect to the store, and do something simple like a head request for a file which is knows is in the store to make sure it works end to end
